docs/_includes/top_nav_home.html

```diff
@@ -1,44 +1,34 @@
 <nav class="navbar home navbar-expand-lg">
-		<div class="container">
-				<div class="s-badge mr-auto">
-						<p class="badge-title"><i class="fas fa-vial"></i><em>Beta</em></p>
-				</div>
-		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-			<span class="navbar-toggler-icon"><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span> </span>
-		</button>
-	
-		<div class="collapse navbar-collapse" id="navbarSupportedContent">
-			<ul class="navbar-nav ml-auto">
-					{% assign links = site.data.navigation.main %}
-					{% for link in links %}
-							{% assign class = nil %}
-							{% if page.url contains link.url %}
-								{% assign class = 'active' %}
-							{% endif %}
-							{% if link.sublinks %}
-								<li class="dropdown {{ class }}">
-									<a href="{{ site.baseurl }}{{ link.url }}" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{{ link.title }} <span class="caret"></span></a>
-									<ul class="dropdown-menu arrow_box">
-										{% for sublink in link.sublinks %}
```

Comment:
Random but can be included in this PR, the GitHub link in the navbar, I think it should trigger the opening of a new tab, instead of taking users away from the secretless page. What do you think?

Reply:
yea. External links should open in a new tab.